src/endpoint/s3/s3_errors.js

```diff
@@ -266,7 +266,7 @@ S3Error.MalformedPOSTRequest = Object.freeze({
 });
 S3Error.MalformedXML = Object.freeze({
     code: 'MalformedXML',
-    message: 'This happens when the user sends malformed xml (xml that doesn\'t conform to the published xsd) for the configuration. The error message is, "The XML you provided was not well-formed or did not validate against our published schema."',
+    message: 'The XML you provided was not well-formed or did not validate against our published schema.',
```

Comment:
You can save what was deleted as a comment (in case it helped someone).
```suggestion
   // This happens when the user sends malformed XML (XML that doesn't conform to the published xsd) for the configuration
    message: 'The XML you provided was not well-formed or did not validate against our published schema.',
```